docs/src/terminology.md

```diff
@@ -170,6 +186,10 @@ A computer participating in a [cluster](terminology.md#cluster).
 
 The number of [validators](terminology.md#validator) participating in a [cluster](terminology.md#cluster).
 
+## nonce account
+
+When creating a transaction, [it must reference a recent hash that is less than 2 minutes old](https://github.com/solana-labs/solana/blob/c7a2fbe7eba18772266f3b4b02ccfb66353ffa3e/sdk/src/clock.rs#L30-L36). To prolong this limit when performing offline signing, or collecting enough signatures to satisfy a multisignature authority, a user may use a [nonce account](https://docs.solana.com/offline-signing/durable-nonce). This account persists on chain, must be rent exempt, and stores a single use nonce. A nonce can be changed by the account’s authority at any moment, and will be automatically changed to the next nonce upon use.
```

Comment:
How about, "an account that stores a blockhash that can only be used once. Use a nonce account to cache a blockhash for use after expiration, such as when performing offline signing."?